src/tests/multihost/ad/test_hostkeytabrotation.py

```diff
@@ -12,27 +18,29 @@
 @pytest.mark.keytabrotation
 class TestHostKeytabRotation(object):
     """ Keytab Rotation Test cases
-    @setup:
-    1.  Configure RHEL client to join to Windows AD using
+    :setup:
+      1. Configure RHEL client to join to Windows AD using
         realm
-    2. set machine password age to 1 day in sssd.conf
+      2. set machine password age to 1 day in sssd.conf
+    :casecomponent: sssd
```

Comment:
casecomponent, requirement these two are already mentioned beginning of the script . 
And these are exactly same . 

I think  these two parameters should be mentioned only once and it should by beginning of script .  

i also did not understand the  "upstream" parameter.  Can we set a default value for  (https://github.com/SSSD/sssd)
Its always "yes"